templates/create-auto-scaling-group.template

```diff
@@ -0,0 +1,656 @@
+        },
+        "AsgHandlerFunctionName": {
+            "Type": "String",
+            "Description": "The Autoscale handler Lambda function name."
+        },
+        "AsgHandlerFunctionArn": {
+            "Type": "String",
+            "Description": "The ARN of the Autoscale handler Lambda function."
+        },
+        "AsgHealthCheckGracePeriod": {
+            "Type": "Number",
+            "Default": 300,
+            "MinValue": 60,
+            "ConstraintDescription": "must be a valid number not less than 60.",
+            "Description": "The length of time (in seconds) that Auto-Scaling waits before checking an instance's health status. Minimum is 60."
+        },
+        "AsgScaleInThreshold": {
+            "Type": "Number",
+            "Default": 25,
+            "MinValue": 1,
+            "MaxValue": 100,
+            "ConstraintDescription": "must be a valid number between 1 and 100.",
+            "Description": "The threshold (in percentage) for the FortiGate Auto-Scaling group to scale-in (remove) 1 instance. Minimum is 1. Maximum is 100."
+        },
+        "AsgScaleOutThreshold": {
+            "Type": "Number",
+            "Default": 80,
+            "MinValue": 1,
+            "MaxValue": 100,
+            "ConstraintDescription": "must be a valid number between 1 and 100.",
+            "Description": "The threshold (in percentage) for the FortiGate Auto-Scaling group to scale-out (add) 1 instance. Minimum is 1. Maximum is 100."
+        },
+        "AsgDesiredCapacity": {
+            "Type": "Number",
+            "Default": 0,
+            "MinValue": 0,
+            "ConstraintDescription": "must be a valid number not less than 1.",
+            "Description": "The number of FortiGate instances the group should have at any time, also called desired capacity. Must keep at least 2 FortiGates in the group for High Availability. Minimum is 2."
+        },
+        "AsgMinSize": {
+            "Type": "Number",
+            "Default": 0,
+            "MinValue": 0,
+            "ConstraintDescription": "must be a valid number not less than 1.",
+            "Description": "Minimum number of FortiGate instances in the Auto-Scaling Group. Minimum is 2."
+        },
+        "AsgMaxSize": {
+            "Type": "Number",
+            "Default": 0,
+            "MinValue": 0,
+            "ConstraintDescription": "must be a valid number not less than 1.",
+            "Description": "Maximum number of FortiGate instances in the Auto-Scaling Group. Minimum is 2."
+        },
+        "AsgCooldown": {
+            "Type": "Number",
+            "Default": 300,
+            "MinValue": 60,
+            "MaxValue": 3600,
+            "ConstraintDescription": "must be a valid number between 60 and 3600.",
+            "Description": "Auto-Scaling group waits for the cooldown period (in seconds) to complete before resuming scaling activities. Minimum is 60. Maximum is 3600."
```

Comment:
"Description": "The Auto Scaling group waits for the cooldown period (in seconds) to complete before resuming scaling activities. Minimum is 60. Maximum is 3600."